tests/console/yast2_nfs_client.pm

```diff
@@ -59,14 +59,23 @@ sub run() {
     #
     # Check the result
     #
-    script_run "cat /tmp/nfs/client/file.txt > /dev/$serialdev", 0;
-
+    # check if nfs is mounted
+    script_run 'mount|grep nfs';
     # Wait for more than 90 seconds due to NFSD's 90 second grace period.
-    wait_serial('It worked', 100) || die "Reading from nfs failed.";
+    diag 'waiting 90 second due NFS grace period';
+    sleep 90;
+    # script_run is using bash return logic not perl logic, 0 is true
+    if ((script_run 'grep "It worked" /tmp/nfs/client/file.txt') != 0) {
```

Comment:
for me script_run(p1, p2) looks like part of script_run, (script_run p1, p2) is obvious left side of condition